docs/reference/ml/anomaly-detection/apis/ml-api.asciidoc

```diff
@@ -4,7 +4,7 @@
 == {ml-cap} {anomaly-detect} APIs
 
 You can use the following APIs to perform {ml} {anomaly-detect} activities. See
```

Comment:
> See ... advanced job configuration options in Kibana.

We're removing the separate definitions pages for the ML APIs, so I think it's better to just remove this sentence.  Thanks for catching this!

Reply:
Removed with 3b0f6f383cda1bac6793a28d5bbca60020a15a48. Thanks @lcawl!